lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
This is not correct, `__attribute__ ((aligned()))` is specified in bytes, not bits. Also, a structure will be aligned to the largest alignment of any member, which is 64 bits / 8 bytes anyway if there is a pointer or int64 in it - so this seems redundant?